pages/resources.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center mb-6">Resources</h1>
+    <p class="mb-3 underline hover:text-gray-500">
+      <a href="arcgis_online_files.zip" download
+        >Download zipfile of CSV files for ArcGIS Online</a
```

Comment:
This page feels a bit sparse. Would it be worth adding either some text at the top saying what this page offers and then providing the links as a list or sectioning each link and adding a bit of supporting text saying what the resource is? Cheers Ruby

Reply:
have added in an introduction sentence and a descripting sentence for each source. Have also deleted the paragraph and link styling as this is now done globally so will appear as such when merged in.

Reply:
ahh cool. Thanks Ruby. I'll keep this issue open and once we've merged in the link and paragraph styles, I'll take another look at this to make sure all is ok but seems like the correct approach. Thank you!

Reply:
paragraphs usually have a bottom margin so there is a gap between paragraphs. Are we able to add this please? Otherwise, looks good. Thanks Ruby

Reply:
oh had removed mb-3 to p in the basic layout but strangely  doesn't seem to be working, so will add it back in

Reply:
thanks Ruby. Looks good now. Merging in